code/player/controller/DefaultWalkController.cs

```diff
@@ -12,6 +12,15 @@ public partial class DefaultWalkController : WalkController
         public float MaxSprintSpeed = 300f;
         public float StaminaLossPerSecond = 30f;
         public float StaminaGainPerSecond = 25f;
+        public float FallDamageVelocity = 550f;
+        public float FallDamageScale = 0.25f;
+        public bool IsDiving = false;
+        public float DrownDamageTime = 15f;
+        public float DrownDamagePerSecond = 10f;
+        public TimeSince TimeSinceDivingStarted = 0f;
```

Comment:
Just suggesting maybe some potential renames

```csharp
IsDiving => IsUnderwater // To avoid confusion with the action of diving itself.
DrownDamageTime => DrowningDuration? // This one is a bit more difficult.
TimeSinceDivingStarted => TimeSpentUnderwater/TimeUnderwater/DurationUnderwater
```